src/components/bookForms.js

```diff
@@ -34,42 +35,33 @@ const Forms = () => {
       setCategory('');
     }
   };
-
-  const Selectbk = [{
-    value: '', label: 'category', disabled: true, selected: true,
-  },
-  { value: 'Action', label: 'Action' },
-  { value: 'Documentary', label: 'Documentary' },
-  { value: 'History', label: 'History' },
-  { value: 'Science', label: 'Science' },
-  { value: 'Medicine', label: 'Medicine' },
-  ];
   return (
-    <form onSubmit={eventSubmit} className="inputfield">
-
-      <input type="text" placeholder="Book Title" required id="title" name="title" value={title} onChange={eventTitle} />
+    <div className="addSection">
+      <hr />
+      <div className="addHeader">Add New Book</div>
+      <div className="input-cont">
+        <form onSubmit={eventSubmit} className="form-cont">
+          <input type="text" className="input-field" placeholder="Book Title" required id="title" name="title" value={title} onChange={onChange} />
 
-      <input type="text" placeholder="Book Author" required id="author" name="author" value={author} onChange={eventAuthor} />
+          <input type="text" className="input-field" placeholder="Book Author" required id="author" name="author" value={author} onChange={onChange} />
 
-      <select
-        name="category"
-        id="category"
-        value={category}
-        onChange={eventCategory}
-      >
-        {Selectbk.map((category) => (
-          <option
-            key={category.value}
-            value={category.value}
-            disabled={category.disabled}
-            selected={category.selected}
+          <select
+            name="category"
+            id="category"
+            className="input-cat"
+            value={category}
+            onChange={onChange}
           >
-            {category.label}
-          </option>
-        ))}
-      </select>
-      <button type="submit">Add Book</button>
-    </form>
+            <option value="Category">Category</option>
+            <option value="Action">Action</option>
+            <option value="Romance">Romance</option>
+            <option value="Horror">Horror</option>
+            <option value="Adventure">Adventure</option>
+          </select>
+          <button type="submit" className="submit-btn">Add Book</button>
+        </form>
+      </div>
+    </div>
```

Comment:
- [ ] It would be nice if you could stretch the input fields to align with the cards from left to right so your design matches the original design. Kindly modify your styles 👍🏽

<img width="1700" alt="Screenshot 2023-05-27 at 10 13 42 PM" src="https://github.com/emmanuelsteven/BookStore-React-Redux/assets/50941074/38e84b73-5531-401a-a6d6-e56070e385f4">